tutorials/export/exporting_for_android.rst

```diff
@@ -29,19 +29,19 @@ Download and install the Android SDK.
   - Ensure that the `required packages <https://developer.android.com/studio/intro/update#recommended>`__ are installed as well.
 
     - Android SDK Platform-Tools version 30.0.5 or later
-    - Android SDK Build-Tools version 30.0.3
-    - Android SDK Platform 33
+    - Android SDK Build-Tools version 34.0.0
+    - Android SDK Platform 34
     - Android SDK Command-line Tools (latest)
     - CMake version 3.10.2.4988404
-    - NDK version r23c (23.2.8568313)
+    - NDK version r26c (26.2.11394342)
```

Comment:
For the `ndk` version, we want to remain at `23.2.8568313` for the time being (the newer versions modify the build process which also requires us to update Godot Android build process, so we're deferring the task for now).

Reply:
Sorry, I didn't realize that the version listed in the list was not the same as the one in the command line, I thought it needed to be updated to the newer.